mri/operators/fourier/non_cartesian.py

```diff
@@ -363,7 +363,7 @@ def op(self, data, *args):
         return self.impl.op(data, *args)
 
     def adj_op(self, coeffs, *args):
-        """ This method calculates inverse masked non-uniform Fourier
+        """ This method computes inverse masked non-uniform Fourier
```

Comment:
(D401) First line should be in imperative mood.
(D210) No whitespaces allowed surrounding docstring text
```suggestion
        """Compute the inverse masked non-uniform Fourier
```